docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '3'
+services:
+  mysql:
+    image: 'mysql'
+    restart: always
+    platform: linux/x86_64
+    volumes:
+      - ./db-data:/var/lib/mysql
+    expose:
+      - '${DB_PORT}'
+    ports:
+      - '${DB_PORT}:${DB_PORT}'
+    environment:
+      MYSQL_ROOT_PASSWORD: '${DB_PASSWORD}'
+      MYSQL_DATABASE: '${DB_NAME}'
+      MYSQL_ROOT_HOST: '%'
+  mysql-test:
+    image: 'mysql'
+    restart: always
+    platform: linux/x86_64
+    volumes:
+      - ./db-test-data:/var/lib/mysql
+    ports:
+      - '${DB_PORT_TEST}:${DB_PORT}'
+    environment:
+      MYSQL_ROOT_PASSWORD: '${DB_PASSWORD}'
+      MYSQL_DATABASE: '${DB_NAME}'
+      MYSQL_ROOT_HOST: "%"
+  # api:
```

Comment:
Are there any benefits in leaving this in?

Reply:
I was getting some errors running the API and didn't finished it, that's why is commented, sorry